docs/snippets/common/storybook-preview-auto-docs-template.mdx.mdx

```diff
@@ -0,0 +1,15 @@
+```mdx
+{/* CustomTemplate.mdx */}
+
+# Get started
```

Comment:
This seems like a weird example to use for a project-level template?

What if we used this to show the default template, which would hint at how to use various Doc Blocks?

Reply:
Rewrite this in MDX: https://github.com/storybookjs/storybook/blob/next/code/ui/blocks/src/blocks/DocsPage.tsx